clients/csrf/csrf_token_manager_test.go

```diff
@@ -166,3 +180,18 @@ func createRequest(method string) *http.Request {
 
 	return request
 }
+
+func RoundTripMock(req *http.Request, transport *Transport) *http.Response {
```

Comment:
This method just maps the request to the response. Just test what has been set in the request and remove this method

Reply:
The methods with capital first letter are public